modules/covid_vaccine/app/models/covid_vaccine/v0/raw_expanded_form_data.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module CovidVaccine
+  module V0
+    class RawExpandedFormData
+      include ActiveModel::Validations
+
+      ATTRIBUTES = %w[email zip_code vaccine_interest birth_date].freeze
+      ZIP_REGEX = /\A^\d{5}(-\d{4})?$\z/.freeze
+
+      attr_accessor(*ATTRIBUTES)
+
+      validates :email, format: { with: URI::MailTo::EMAIL_REGEXP }
+      validates :vaccine_interest, presence: true
+      validates :zip_code, format: { with: ZIP_REGEX, message: 'should be in the form 12345 or 12345-1234' }
+      validates :birth_date, format: { with: /\A\d{4}-\d{2}-\d{2}\z/, message: 'should be in the form yyyy-mm-dd' },
+                             allow_blank: true
```

Comment:
why allow_blank here?

Reply:
ah guessing this is just pulled over from KMI and needs updating per your PR comment :-)